apps/web/src/views/Info/index.tsx

```diff
@@ -32,22 +32,23 @@ export const InfoPageLayout = ({ children }) => {
 
   return (
     <>
-      {chainName === 'BSC' && (
-        <SubMenuItems
-          items={[
-            {
-              label: t('Swap'),
-              href: '/info',
-            },
-            {
-              label: t('StableSwap'),
-              href: '/info?type=stableSwap',
-            },
-          ]}
-          activeItem={isStableSwap ? '/info?type=stableSwap' : '/info'}
-        />
-      )}
-
+      <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'center', justifyContent: 'center' }}>
+        {chainName === 'BSC' && (
```

Comment:
why are we checking on BSC chain?

Reply:
I put that div there just to center the submenu which was generated differently than it was on other pages. That conditional `{chainName == 'BSC' && (` was there before so I kept it, but you're right I don't see why generating the submenu should depend on chain.

Reply:
Maybe is the StableSwap info only supported by Binance smart chain?